### DISCUSSION TITLE
[Feature Request] Pinchflat YouTube media manager

### DISCUSSION THREAD

**Discussion by crispybegs** [gethomepage/homepage#3912] [Feature Requests]
### Description

would love a widget for the Pinchflat downloader - https://github.com/kieraneglin/pinchflat
imo one of the best out there, made by @kieraneglin

**Comment by shamoon** [comment]
I’m always nervous adding projects that are this young, many don’t stick around too long…

**Reply by crispybegs** [reply]
yes fair point, i get that

**Reply by kieraneglin** [reply]
Yeah, fair enough! Maybe we'll come back to it next year (:

**Reply by Mondrethos** [reply]
Hi @shamoon, is it possible for this to be considered? It's the new year and Pinchflat has become a popular alternative to tube archivist.

**Reply by dsmith-rldatix** [reply]
I would also like to see this added. It's a great project building a strong following and actively maintained.

**Reply by 9r2a5m70** [reply]
I would like to see this added as well.

**Reply by bleeblonks** [reply]
Still using this actively at this time and would love to see as well. No rush just showing my support. Love both of these services you guys provide. Thank you so much.

**Reply by rlee0047** [reply]
Bump to add Pinchflat

**Reply by borski** [reply]
I would love to see this added; I use it regularly

**Reply by rlee0047** [reply]
> Bump to add Pinchflat

I guess bump isn't the correct way to say that, idk...

**Reply by drewski3420** [reply]
Pinchflat doesn't expose any metrics via API, right? Wouldn't that be necessary in order to create a widget?
